StripeFinancialConnections/StripeFinancialConnections/Source/Native/LinkAccountPicker/LinkAccountPickerDataSource.swift

```diff
@@ -39,7 +39,25 @@ final class LinkAccountPickerDataSourceImplementation: LinkAccountPickerDataSour
     private let apiClient: FinancialConnectionsAPIClient
     private let clientSecret: String
     private let consumerSession: ConsumerSessionData
-    let dataAccessNotice: FinancialConnectionsDataAccessNotice?
+    var dataAccessNotice: FinancialConnectionsDataAccessNotice? {
+        var dataAccessNotice: FinancialConnectionsDataAccessNotice?
+        if
+            let networkedAccountsResponse,
+            let returningNetworkingUserAccountPicker = networkedAccountsResponse.display?.text?.returningNetworkingUserAccountPicker,
+            !returningNetworkingUserAccountPicker.accounts.isEmpty
+        {
+            if selectedAccounts.isEmpty {
+                dataAccessNotice = returningNetworkingUserAccountPicker.accounts.first?.dataAccessNotice
```

Comment:
[link to Stripe.js code](https://git.corp.stripe.com/stripe-internal/stripe-js-v3/blob/15a677fc/src/linkedAccounts/inner/components/panes/v3/ConsentPane/content/DataAccessNoticeDrawer/useDataAccessNoticeText.ts#L51)